README.rst

```diff
@@ -1,17 +1,158 @@
+
+********
+Features
+********
+
+Feature: Pages and Resources Studio Tab
+=======================================
+
+Enables a "Pages & Resources" menu item in Studio, under the "Content" menu.
+
+Requirements
 ------------
 
+The following are external requirements for this feature to function correctly:
+
+* ``edx-platform`` Django settings:
+
+  * ``COURSE_AUTHORING_MICROFRONTEND_URL``: must be set and point to this MFE's deployment URL.
+
+* ``edx-platform`` Waffle flags:
+
+  * ``discussions.pages_and_resources_mfe``: must be enabled for the set of users meant to access this feature.
+
+* `frontend-app-learning <https://github.com/openedx/frontend-app-learning>`_: This MFE expects it to be the LMS frontend.
+* `frontend-app-discussions <https://github.com/openedx/frontend-app-discussions/>`_: This is what the "Discussions" configuration provided by this feature actually configures.  Without it, discussion settings are ignored.
+
+Configuration
+-------------
+
+In additional to the standard settings, the following local configuration items are required:
+
+* ``LEARNING_BASE_URL``: points to Learning MFE; necessary so that the `View Live` button works
+* ``ENABLE_PROGRESS_GRAPH_SETTINGS``: allow enabling or disabling the learner progress graph course-wide
+
+Feature Description
+-------------------
+
+Clicking on the "Pages & Resources" menu item takes the user to the course's ``pages-and-resources`` standalone page in this MFE.  (In a devstack, for instance: http://localhost:2001/course/course-v1:edX+DemoX+Demo_Course/pages-and-resources.)
+
+UX-wise, **Pages & Resources** is meant to look like a Studio tab, so reproduces Studio's header.
+
+For a particular course, this page allows one to:
+
+* Configure the new Discussions MFE (making this a requirement for it).  This includes:
+
+  * Enabling/disabling the feature entirely
+  * Picking a different discussion provider, while showing a comparison matrix between them:
+
+    * edX
+    * Ed Discussion
+    * InScribe
+    * Piazza
+    * Yellowdig
+
+  * Allowing to configure the selected provider
+
+* Enable/Disable learner progress
+* Enable/Disable learner notes
+* Enable/Disable the learner wiki
+* Enable/Disable the LMS calculator
+* Go to the textbook management page in Studio (in a devstack: http://localhost:18010/textbooks/course-v1:edX+DemoX+Demo_Course)
+* Go to the custom page management page in Studio(in a devstack http://localhost:18010/tabs/course-v1:edX+DemoX+Demo_Course)
+
+Feature: New React XBlock Editors
+=================================
+
+This allows an operator to enable the use of new React editors for the HTML, Video, and Problem XBlocks, all of which are provided here.
+
+Requirements
+------------
+
+* ``edx-platform`` Django settings:
+
+  * ``COURSE_AUTHORING_MICROFRONTEND_URL``: must be set and point to this MFE's deployment URL
```

Comment:
Would be possible to be more exact that `COURSE_AUTHORING_MICROFRONTEND_URL` setting need to be define in `cms` part of `edx-platform`

While testing for this mfe earlier, I had to to do the following as well, those aren't direclty related to this MFE rather depends on the deployment type, but I am not sure if we should have them here or in tutor-mfe. 

- `ENABLE_CORS_HEADERS` set this to true in cms _This because this MFE is going to make API call to the studio, and given its mostly going to live on different subdomain that the studio, i.e. in tutor apps.*_
- `CORS_ORIGIN_WHITELIST`, `LOGIN_REDIRECT_WHITELIST`, and `CSRF_TRUSTED_ORIGINS` _add domain or subdomain of this mfe_

Reply:
Nice catches, @ghassanmas.  I think specifying that the URL needs to be set in the Studio env makes sense here.

The other stuff will likely make it into the changes to `tutor-mfe` plugin, but still, it's good that you listed it for us!  I'll make a reference in [the corresponding ticket](https://github.com/overhangio/tutor-mfe/issues/79).